tests/test_server_basic.py

```diff
@@ -221,6 +220,40 @@ def test_valid_token(dispatcher_live_fixture):
     logger.info(json.dumps(jdata, indent=4))
 
 
+def test_download_products(dispatcher_live_fixture, empty_products_files_fixture):
+    server = dispatcher_live_fixture
+
+    logger.info("constructed server: %s", server)
+
+    session_id = empty_products_files_fixture['session_id']
+    job_id = empty_products_files_fixture['job_id']
+
+    params = {
+            # since we are passing a job_id
+            'query_status': 'ready',
+            'file_list': 'test.fits.gz.recovered',
+            'download_file_name': 'output_test',
+            'session_id': session_id,
+            'job_id': job_id
+        }
+
+    c = requests.get(server + "/download_products",
+                     params=params)
+
+    print("content:", c.text.encode())
+
+    assert c.status_code == 200
```

Comment:
c.content is bytes of response, it can be inspected and compared with ` empty_products_files_fixture['content']`

Reply:
that is actually some test code, that serves no purpose for this test

Reply:
ok, but do you test the result of the request?

Reply:
Yes, it check that the content returned matches

Reply:
But where is the returned content?